global-script.lua

```diff
@@ -145,6 +145,14 @@ explorerDamage = "574835"
 dahanHealth = "746488"
 dahanDamage = "d936f3"
 -----
+secondHandDiscardPositions = {
```

Comment:
rather than hardcode some vectors here, you should be able to pull up this value programmatically `Player[color].getHandTransform(2).position`

Reply:
Done.